docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -465,3 +467,118 @@ The following overrides are available:
   For more information, see [Helm chart: Prerequisites](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#prerequisites).
 
 - Overrides to use a secret for LDAP authentication. Refer to [Create secrets for Kubernetes](../../../../secure/authentication/ldap-authentication-ysql/#create-secrets-for-kubernetes).
+
+## Configure Multi-Cluster Services
+
+If you plan to create multi-region YugabyteDB universes, you can setup [Multi-Cluster Services](https://git.k8s.io/enhancements/keps/sig-multicluster/1645-multi-cluster-services-api) (MCS) across your Kubernetes clusters. This section covers implementation specific details for setting up MCS on various cloud providers and service mesh tools.
+
+{{< warning title="MCS support is in beta" >}}
+
+MCS support in YugabyteDB Anywhere is currently in the beta state. It comes with following caveats:
+- The metrics screen of universe might not show correct metrics for all the pods.
+- xCluster replication needs manual intervention to work on OpenShift MCS.
+- There can be some untested combinations and edge cases which might have bugs.
+
+The Kubernetes Multi-Cluster Services API is currently in alpha state, though there are various implementation of MCS which are [considered to be stable](https://github.com/kubernetes-sigs/mcs-api/issues/17#issuecomment-1309073682). To know more, see [API versioning](https://kubernetes.io/docs/reference/using-api/#api-versioning).
+
+{{< /warning >}}
+
+### Configure GKE Multi-Cluster Services
```

Comment:
should we call these three sections 

### Prepare your kubernetes cluster for GKE MCS